### PR TITLE
Allow CSS on elements

### PR DESCRIPTION
**Pull request by spotby**
I added support for adding a a css className, and I had to switch the rendered element to a div, so that I could apply borders to the input field. My use of this in my app is as follows:

before fix:
image

after fix:
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
     "eslint-config-stripe": "^1.0.10",
     "eslint-plugin-flowtype": "^2.32.1",
     "eslint-plugin-jest": "^19.0.1",
+    "eslint-plugin-jsx-a11y": "2.2.3",
```
Comment (michelle) [inline package.json]: Hm, what's this change?

Comment (spotby) [inline package.json]: i couldn't get my local eslint working with atom with the 2.x.x version (related to -> (https://github.com/evcohen/eslint-plugin-jsx-a11y/issues/116)  . May have been my local setup. feel free to remove it.

**Comment by michelle** [comment]
Thanks for the PR + screenshots!

Our rationale for making the wrapper a span is that HTML input elements usually aren't block-level elements. We generally recommend that users style the `.StripeElement` class as they would their HTML input elements, so if we made this a div, users would need to style them differently.

Maybe an alternative fix here is to allow `*Element`s to take a `className`, which would be applied to the wrapping `span`?

I'm curious what you think.

**Comment by michelle** [comment]
Additionally, Elements automatically adds focus/invalid/empty/complete classnames to its wrapping HTML element. If the user were to instead style the HTML element that wraps our React wrapper, they wouldn't get those classes.

**Comment by spotby** [comment]
huh.. I used the .StripeElement to skin it and couldn't figure out how to put a border around it. my css was:
```
.StripeElement {
  background-color: white;
  padding: 8px 12px;
  border-radius: 4px;
  border: 1px solid black;
  box-shadow: 0 1px 3px 0 #e6ebf1;
  -webkit-transition: box-shadow 150ms ease;
  transition: box-shadow 150ms ease;
}
```

**Comment by michelle** [comment]
Hm, that border didn't work? 😲

**Comment by michelle** [comment]
(If you run the demo, it does the exact same thing: https://github.com/stripe/react-stripe-elements/blob/master/demo/index.html#L70-L78)

File: package.json (hunk 1 above)
Comment (michelle) [inline package.json]: :+1: Thanks for the context!

**Comment by spotby** [comment]
ok. it looks like I was missing a display:block in my css. Whoops. sorry for the trouble!

**Comment by tim-soft** [comment]
I think that adding className on elements would be very helpful

**Pull request closed without merging**